compiler/src/main/java/com/squareup/anvil/compiler/ModuleMerger.kt

```diff
@@ -92,112 +84,107 @@ internal class ModuleMerger(
         // modules contain the same bindings and are contributed to the same scope. To avoid this
         // issue we filter all generated Anvil modules except for the one that was generated for
         // this specific class.
-        val fqName = it.fqNameSafe
+        val fqName = it.fqName
         !fqName.isAnvilModule() || fqName == anvilModuleName
```

Comment:
You can inline the variable now, because we cache the value:
```suggestion
        !it.fqName.isAnvilModule() || it.fqName == anvilModuleName
```